src/components/AlignmentHost/AlignmentHost.ts

```diff
@@ -69,30 +68,30 @@ namespace fabric {
 
     private _openalignment(): void {
       this._renderAlignment();
-      
+
       // this._savealignmentSize();
       this._findAvailablePosition();
       this._showalignment();
 
       // Delay the click setting
       setTimeout( () => { this._setDismissClick(); }, 100);
     }
-    
-    private _getAlignmentSize() {
-      switch(this._direction) {
-        case "left":
-          
-          break;
-        case "right":
-          break;
-        case "top":
-          break;
-        case "bottom":
-          break;
-        default:
-        this._setPosition();
-      }
-    }
+
+    // private _getAlignmentSize() {
```

Comment:
I'm assuming this will be either revised or removed at some point--we shouldn't leave commented-out blocks of code in PRs if possible.